src/chrome/content/rules/WP-Engine.xml

```diff
@@ -69,7 +70,19 @@
 		<test url="http://www.wpengine.com/" />
 		<test url="http://www.wpengine.com/blog" />
 
-
+	<!-- deeper domains cause certificate mismatch error -->
+	<exclusion pattern="^http://([\w.-]+)\.([\w-]+)\.wpengine\.com"/>
+
+		<test url="http://talkdesk.staging.wpengine.com/"/>
+		<test url="http://kelvin.staging.wpengine.com/"/>
+		<test url="http://cfps.staging.wpengine.com/"/>
+		<test url="https://talkdesk.staging.wpengine.com/"/>
+		<test url="https://kelvin.staging.wpengine.com/"/>
+		<test url="https://cfps.staging.wpengine.com/"/>
+		
+	<!-- https links from other pages to these will cause MCB for important elements, hence the downgrades -->
```

Comment:
To be honest, I'm not familiar with wpengine.com. Now that I think of it, judging by the "staging" keyword, I don't think other pages would link to `*.*.wpengine.com`. I removed the downgrade rule.